assets/templates/partials/gtm-data-layer.tmpl

```diff
@@ -35,6 +35,14 @@ dataLayer = [{
         "numberOfResults": {{.Count}},
         "resultsPage": {{.Pagination.CurrentPage}},
     {{ end }}
+    {{ if eq .Type "ReleasePage" }}
+        "release-status": {{.ReleasePageData.ReleaseStatus}},
+        "release-date": {{.ReleasePageData.ReleaseDate}},
```

Comment:
What's the difference between this release-data/releaseDate than the one on line `31`?

Also, is the date format correct?